### PR TITLE
Minor change on docs

### PR DESCRIPTION
**Pull request by MarceloSalazar**
Just a few tweaks to have consistency with other Mbed OS examples

@Patater

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,18 +9,22 @@ List of examples contained within this repository:
 ## Prerequisites
 * Install <a href='https://github.com/ARMmbed/mbed-cli#installing-mbed-cli'>Mbed CLI</a>
 
-## Deploy
-The following are the steps required for deployment:
-* Clone this repository: `git clone git@github.com:ARMmbed/mbed-os-example-mbed-crypto.git`
+## Import
+The following are the steps required for install the application:
```
Comment (Patater) [inline README.md]: Recommend changing "for" to "to".

```diff
- the steps required for install the application
+ the steps required to install the application
```

**Review by Patater — CHANGES_REQUESTED** [review]

**Review by itayzafrir — CHANGES_REQUESTED** [review]
Aside from the comment @Patater left, please fix the redundancy between the `Program your board` and `Run` sections.

**Comment by MarceloSalazar** [comment]
Fixed. Hopefully is now ok.

**Comment by MarceloSalazar** [comment]
Let's also update the mbed-os-5.11.0-oob branch

File: README.md [hunk 2]
```diff
@@ -10,24 +9,24 @@ List of examples contained within this repository:
...
-* Fetch Mbed OS: `mbed deploy`
 
 ## Compile
 To compile the example program use `mbed compile` while specifying the target platform and the compiler.
 For example, in order to compile using the ARM GCC compiler and a K64F target platform use: `mbed compile -m K64F -t ARM`.
 
 Once the compilation is completed successfully a binary file will be created: `./BUILD/K64F/GCC_ARM/mbed-os-example-mbed-crypto.bin`
 
+## Program your board
+1. Connect your mbed device to the computer over USB.
```
Comment (itayzafrir) [inline README.md]: Just for the sake of consistency, maybe capitalize `mbed`
```suggestion
1. Connect your Mbed device to the computer over USB.
```

**Review by itayzafrir — COMMENTED** [review]
One last tweak and then we're good.

**Review by itayzafrir — APPROVED** [review]

**Review by Patater — APPROVED** [review]

**Pull request merged**